src/utils/validations.ts

```diff
@@ -10,7 +10,7 @@ export const memberCodeValidations = [
   (val: string) =>
     val.length === 10 || 'El codigo debe tener una longitud de 10 caracteres',
   (val: string) =>
-    val.match(/^[A-Z]{2}[0-9]{8}$/i) || 'El formato del codigo es incorrecto',
+    !!val.match(/^[A-Z]{2}[0-9]{8}$/i) || 'El formato del codigo es incorrecto',
```

Comment:
It seems like it can sometimes output `[""]`, which is truthy. Please take care of that special case

![image](https://user-images.githubusercontent.com/36519478/182327646-89fb7ae3-13e1-41c5-a143-a0760e275dc8.png)

Reply:
I tested and not found that match returns an empty Array

![LastRecord](https://user-images.githubusercontent.com/66505715/183271395-768ffc19-f0ba-47e4-9c76-4d40e5f318b9.gif)

Just in case that the argument was empty

![image](https://user-images.githubusercontent.com/66505715/183271414-b649505e-7e85-4a3a-9f13-c38307d64f92.png)

But is not possible because in this case the argument is static (is always a regexp which don't change dynamically)